lib/teecket/selectors/malindo_air.rb

```diff
@@ -28,7 +28,7 @@ def destination_selector(elem)
 
     def depart_arrive_at_formatter(datetime)
       DateTime
-        .strptime(datetime.gsub(%r(^\/Date\(|\)\/), ""), "%Q")
+        .strptime(datetime.gsub(%r{^\/Date\(|\)\/}, ""), "%Q")
```

Comment:
Do not use `DateTime.strftime` without zone. Use one of `Time.zone.strftime`, `DateTime.strftime.current`, `DateTime.strftime.in_time_zone`, `DateTime.strftime.utc`, `DateTime.strftime.getlocal`, `DateTime.strftime.iso8601`, `DateTime.strftime.jisx0301`, `DateTime.strftime.rfc3339`, `DateTime.strftime.to_i`, `DateTime.strftime.to_f` instead.